Atomic/util.py

```diff
@@ -119,7 +125,10 @@ def default_ro_container_context():
 
 def write_out(output, lf="\n"):
     sys.stdout.flush()
-    sys.stdout.write(str(output) + lf)
+    if is_python2:
+        sys.stdout.write(output.encode('utf-8') + lf)
+    else:
+        sys.stdout.write(output + str(lf))
 
```

Comment:
BC in python 3, output is a str and lf is a byte and then cannot be concatenated.